test/test_calculus.py

```diff
@@ -29,5 +29,15 @@ def test_calculus():
             "{x->1.51213}",
             "Issue #1235",
         ),
+        (
+            "v1 := Exp[y] - 3y == 0; v2 = {y, 2}; FindRoot[v1, v2]",
+            "{y->1.51213}",
+            "Issue #1235",
+        ),
+        (
+            "v1 := Exp[z] - 3z == 1.0; v2 = {z, 3}; FindRoot[v1, v2]",
```

Comment:
why these two different equations should have the same root?

Reply:
> As far as I can understand, FindRoot[f[x]==g[x],{x,x0}] tries to get a solution of the equation
> f[x]-g[x]==0 around x==x0, which is the same than
> 
> FindRoot[f[x+x0]==g[x+x0],{x,0}]+x0
> 
> but this is not what we have here I think. Am I wrong?

Yes, you are correct and this code is incorrect (except in the trivial but often case where the right-hand side of the `==` is 0. 

So this code is wrong.